### DISCUSSION TITLE
Does 'terminal' in sections in dashboard cache the output?

### DISCUSSION THREAD

**Discussion by redoxahmii** [folke/snacks.nvim#173] [Q&A]
might be a dumb question but does using `section = "terminal"` cache the output of the command we are using in the dashboard?
I run a `cat` command to display some information on the pane and it does not refresh if i change something inside it and i would like that behavior not to happen.

**Comment by redoxahmii** [comment]
i have also tried with using `glow` as those files are markdown files but that takes a lot of time to run and display anything.

**Comment by folke** [comment]
It is and it's all in the docs. See here https://github.com/folke/snacks.nvim/blob/main/docs/dashboard.md#snacksdashboardsectionsterminal

You can change the `ttl` or set it to `0` to disable caching

**Reply by redoxahmii** [reply]
missed it thought `ttl` was for something different.
Thanks!
